test/e2e/framework/kcp.go

```diff
@@ -89,7 +89,7 @@ func TestServerWithClientCAFile(clientCAFile string) []string {
 // start a test server with the given token auth file.
 func TestServerArgsWithTokenAuthFile(tokenAuthFile string) []string {
 	return []string{
-		"-v=4",
```

Comment:
this is undesired, isn't it?

Reply:
It's for debugging the failures

Reply:
for debugging sure, but to merge? We already went from 2 to 4 sometime ago. Now 5?

Reply:
Not to merge, no